File: compiler-group-traslator/src/test/resources/c/IfFalseSingleElseEmpty.c
#include<stdio.h>
#include<stdlib.h>
#include<stdbool.h>
#include "GC.c"
#include "ExceptionHandlingFunctions.c"
VM* vm;

int main(char* args) {
	vm = newVM();
	int a;

	a = 0 ;
	if (false) {
		a = 1 ;
	}else{
	}
	assertTrue(a == 0);
	return 0;
}

File: compiler-group-traslator/src/test/resources/IfFalseSingleElseEmpty.java
public class IfFalseSingleElseEmpty {
    public static void main(String[] args) {

        int a;
        a = 0;
        if (false) {
            a = 1;
        } else {
        }
        StaticJavaLib.assertTrue(a == 0);
    }
}
